app/Filament/Resources/ContactFormResource.php

```diff
@@ -120,6 +120,11 @@ public static function table(Table $table): Table
             ]);
     }
 
+    public static function getNavigationBadge(): ?string
+    {
+        return (string)static::$model::where("status", ContactFormStatus::Unread)->count();
+    }
```

Comment:
I think this method will never return `null`. 
So nullable return type (?) is not needed.